src/featureProvider.ts

```diff
@@ -28,7 +28,7 @@ export class ConfigurationMapFeatureFlagProvider implements IFeatureFlagProvider
     }
     async getFeatureFlag(featureName: string): Promise<FeatureFlag | undefined> {
         const featureConfig = this.#configuration.get<FeatureManagementConfiguration>(FEATURE_MANAGEMENT_KEY);
-        return featureConfig?.[FEATURE_FLAGS_KEY]?.find((feature) => feature.id === featureName);
+        return featureConfig?.[FEATURE_FLAGS_KEY]?.findLast((feature) => feature.id === featureName);

```

Comment:
Interesting, this makes more sense to me. 
Design change? AFIAK .NET provider takes the first. Will .NET also change or only FM js/py has a different behavior?

Reply:
This design change is for all languages.